docs/monoDrive_home/Weather.md

```diff
@@ -208,9 +215,36 @@ the array:
 
 ## Using the Weather in the Python Client
 
-TBD
+The [monoDrive Python Client](../../python_client/quick_start) reads weather 
+settings from input JSON files and sends them to the simulator. In the following 
+example, simply place the desired weather configuration into the file 
+`config/weather.json` and construct the simulator:
+
+```python
+ # Construct simulator from file
+simulator = Simulator.from_file(
+    config='./config/simulator_closed_loop.json',
+    trajectory='./trajectories/Closed_Loop.json',
+    sensors='./config/all_sensors.json',
+    weather='./config/weather.json',
+    ego='./config/vehicle.json',
+)
+```
 
 ## Using the Weather in the C++ Client
 
-TBD
-
+The [monoDrive C++ Client](../../cpp_client/cpp_quick_start) reads weather 
+settings from input JSON files and sens them to the simulator. In the following
```

Comment:
JSON files and **sends** them to the simulator.

Reply:
Fixed.